app/controllers/profiles_controller.rb

```diff
@@ -0,0 +1,25 @@
+class ProfilesController < ApplicationController
+  before_action :require_authentication
+
+  def show; end
+
+  def update
+    return unless current_user?
+
+    @user = current_user
+    if @user.update(profile_params)
+      flash[:success] = 'Your profile has been updated'
```

Comment:
Will this flash ever get shown, given we're returning JSON below?

Reply:
Yes. This gets shown right after the user makes an update and is logged out